core/src/gl/texture.cpp

```diff
@@ -8,6 +8,14 @@
 #include "tangram.h"
 #include "log.h"
 
+// Only JPEG, PNG and GIF
+#define STBI_NO_BMP
+#define STBI_NO_PSD
+#define STBI_NO_TGA
```

Comment:
If they are not supported in JS, supporting them here breaks scene file compatibility because of file extensions.